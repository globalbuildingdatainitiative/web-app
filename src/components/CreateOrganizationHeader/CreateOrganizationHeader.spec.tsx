import { CreateOrganizationHeader } from '@components'
import { MemoryRouter } from 'react-router-dom'
import { Route, Routes } from 'react-router'
import { expect, render, test, screen } from '@test'

test('CreateOrganizationHeader', async () => {
  const { baseElement } = render(
    <MemoryRouter initialEntries={['/']}>
      <Routes>
        <Route path='/' element={<CreateOrganizationHeader />} />
      </Routes>
    </MemoryRouter>,
  )
  expect(await screen.findByTestId('CreateOrganizationHeader')).toBeTruthy()
  expect(baseElement).toMatchSnapshot()
})
