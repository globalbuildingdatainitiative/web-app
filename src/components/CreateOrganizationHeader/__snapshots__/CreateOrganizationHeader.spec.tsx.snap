// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CreateOrganizationHeader 1`] = `
<body>
  <div>
    <style
      data-mantine-styles="true"
    >
      :root{--mantine-font-family: Outfit, sans-serif;--mantine-font-family-headings: Outfit, sans-serif;--mantine-breakpoint-xxl: 120em;}
    </style>
    <style
      data-mantine-styles="classes"
    >
      @media (max-width: 35.99375em) {.mantine-visible-from-xs {display: none !important;}}@media (min-width: 36em) {.mantine-hidden-from-xs {display: none !important;}}@media (max-width: 47.99375em) {.mantine-visible-from-sm {display: none !important;}}@media (min-width: 48em) {.mantine-hidden-from-sm {display: none !important;}}@media (max-width: 61.99375em) {.mantine-visible-from-md {display: none !important;}}@media (min-width: 62em) {.mantine-hidden-from-md {display: none !important;}}@media (max-width: 74.99375em) {.mantine-visible-from-lg {display: none !important;}}@media (min-width: 75em) {.mantine-hidden-from-lg {display: none !important;}}@media (max-width: 87.99375em) {.mantine-visible-from-xl {display: none !important;}}@media (min-width: 88em) {.mantine-hidden-from-xl {display: none !important;}}@media (max-width: 119.99375em) {.mantine-visible-from-xxl {display: none !important;}}@media (min-width: 120em) {.mantine-hidden-from-xxl {display: none !important;}}
    </style>
    <div
      class="m_e615b15f mantine-Card-root m_1b7284a3 mantine-Paper-root"
      data-testid="CreateOrganizationHeader"
      data-with-border="true"
      style="--paper-radius: var(--mantine-radius-lg); --paper-shadow: var(--mantine-shadow-sm); --card-padding: var(--mantine-spacing-lg);"
    >
      <div
        class="m_4081bf90 mantine-Group-root"
        style="--group-gap: var(--mantine-spacing-md); --group-align: center; --group-justify: space-between; --group-wrap: wrap;"
      >
        <h6
          class="m_8a5d1357 mantine-Title-root"
          data-order="6"
          style="--title-fw: var(--mantine-h6-font-weight); --title-lh: var(--mantine-h6-line-height); --title-fz: var(--mantine-h6-font-size);"
        >
          Total Members
        </h6>
        <div
          class="m_3eebeb36 mantine-Divider-root"
          data-orientation="vertical"
          role="separator"
          style=""
        />
        <h6
          class="m_8a5d1357 mantine-Title-root"
          data-order="6"
          style="--title-fw: var(--mantine-h6-font-weight); --title-lh: var(--mantine-h6-line-height); --title-fz: var(--mantine-h6-font-size);"
        >
          Organization Name
        </h6>
        <div
          class="m_3eebeb36 mantine-Divider-root"
          data-orientation="vertical"
          role="separator"
          style=""
        />
        <button
          class="mantine-focus-auto mantine-active m_77c9d27d mantine-Button-root m_87cf2631 mantine-UnstyledButton-root"
          data-size="md"
          data-variant="light"
          style="--button-height: var(--button-height-md); --button-padding-x: var(--button-padding-x-md); --button-fz: var(--mantine-font-size-md); --button-radius: var(--mantine-radius-sm); --button-bg: var(--mantine-color-gray-light); --button-hover: var(--mantine-color-gray-light-hover); --button-color: var(--mantine-color-gray-light-color); --button-bd: calc(0.0625rem * var(--mantine-scale)) solid transparent;"
          type="button"
        >
          <span
            class="m_80f1301b mantine-Button-inner"
            style=""
          >
            <span
              class="m_811560b9 mantine-Button-label"
              style=""
            >
              View Members
            </span>
          </span>
        </button>
        <div
          class="m_3eebeb36 mantine-Divider-root"
          data-orientation="vertical"
          role="separator"
          style=""
        />
        <button
          class="mantine-focus-auto mantine-active m_77c9d27d mantine-Button-root m_87cf2631 mantine-UnstyledButton-root"
          data-size="md"
          data-variant="light"
          style="--button-height: var(--button-height-md); --button-padding-x: var(--button-padding-x-md); --button-fz: var(--mantine-font-size-md); --button-radius: var(--mantine-radius-sm); --button-bg: var(--mantine-color-gray-light); --button-hover: var(--mantine-color-gray-light-hover); --button-color: var(--mantine-color-gray-light-color); --button-bd: calc(0.0625rem * var(--mantine-scale)) solid transparent;"
          type="button"
        >
          <span
            class="m_80f1301b mantine-Button-inner"
            style=""
          >
            <span
              class="m_811560b9 mantine-Button-label"
              style=""
            >
              Add Member
            </span>
          </span>
        </button>
      </div>
    </div>
  </div>
</body>
`;
